mantle/platform/api/openstack/api.go

```diff
@@ -313,15 +291,17 @@ func (a *API) CreateServer(name, sshKeyID, userdata string) (*Server, error) {
 		return nil, fmt.Errorf("waiting for instance to run: %v", err)
 	}
 
-	var floatingip *floatingips.FloatingIP
-	if a.opts.FloatingIPPool != "" {
-		floatingip, err = a.createFloatingIP()
+	var floatingip *networkFloatingIPs.FloatingIP
+	if a.opts.FloatingIPNetwork != "" {
+
```

Comment:
nit: Extra new line -- not an issue but also not needed.

Reply:
Will delete the newline in the rebase.